packages/webdriverio/src/index.ts

```diff
@@ -246,15 +253,15 @@ export default class AxeBuilder {
     // ensure we fail quickly if an iframe cannot be loaded (instead of waiting
     // the default length of 30 seconds)
     const { pageLoad } = await this.client.getTimeouts();
-    this.client.setTimeout({
+    (this.client as WebdriverIO.Browser).setTimeout({
```

Comment:
This fixes a problem with older versions of webdriverio as in newer versions they declare the function with the first parameter as [this: Webdriverio.Browser](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/browser/setTimeout.ts#L40).